types/reveal.js/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Reveal 4.4
+// Type definitions for Reveal 4.4.0
```

Comment:
According to the [guideline](https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library), the patch part is not necessary.